sycl/test/check_device_code/cuda/matrix/matrix-nvptx-tf32-test.cpp

```diff
@@ -137,12 +148,13 @@ int main() {
             get_wi_data(sg, sub_b)[i] =
                 round_to_tf32(get_wi_data(sg, sub_b)[i]);
 
-          //CHECK: tail call { float, float, float, float, float, float, float, float } @llvm.nvvm.wmma.m16n16k8.mma.col.col.tf32(i32 {{.*}}, i32 {{.*}}, i32 {{.*}}, i32 {{.*}}, i32 {{.*}}, i32 {{.*}}, i32 {{.*}}, i32 {{.*}}, float {{.*}}, float {{.*}}, float {{.*}}, float {{.*}}, float {{.*}}, float {{.*}}, float {{.*}}, float {{.*}})
+          // CHECK: tail call { float, float, float, float, float, float, float, float } @llvm.nvvm.wmma.m16n16k8.mma.col.col.tf32(i32 %46, i32 %47, i32 %48, i32 %49, i32 %50, i32 %51, i32 %52, i32 %53, float %45, float %44, float %43, float %42, float %41, float %40, float %39, float %38
           sub_c = joint_matrix_mad(sg, sub_a, sub_b, sub_c);
-          //CHECK: tail call void @llvm.nvvm.wmma.m16n16k16.store.d.col.stride.f32.p1f32(float addrspace(1)* {{.*}}, float {{.*}}, float {{.*}}, float {{.*}}, float {{.*}}, float {{.*}}, float {{.*}}, float {{.*}}, float {{.*}}, i32 16)
-          //CHECK-OPAQUE: tail call void @llvm.nvvm.wmma.m16n16k16.store.d.col.stride.f32.p1(ptr addrspace(1) {{.*}}, float {{.*}}, float {{.*}}, float {{.*}}, float {{.*}}, float {{.*}}, float {{.*}}, float {{.*}}, float {{.*}}, i32 16)
-          joint_matrix_store(sg, sub_c, accD.get_pointer(), N,
-                             layout::col_major);
+          // CHECK: tail call void @llvm.nvvm.wmma.m16n16k16.store.d.col.stride.f32.p0f32(float* %63, float %55, float %56, float %57, float %58, float %59, float %60, float %61, float %62, i32 16
+          // CHECK-OPAQUE: tail call { float, float, float, float, float, float, float, float } @llvm.nvvm.wmma.m16n16k16.load.c.col.stride.f32.p0(ptr %20, i32 16
```

Comment:
Is there a reason you changed these tests to use explicit values instead of wildcards (`{{.*}}`)? If no, could we revert it? Not using wildcards makes the test potentially fragile.

Note that the same seems to apply to other tests in sycl/test/check_device_code/cuda/matrix.

Reply:
The ones which I updated were failing, I replaced the failing values with the appropriate ones by running the corresponding command.